.vscode/launch.json

```diff
@@ -1,292 +0,0 @@
-{
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
```

Comment:
we shouldn't delete this, I hope it was deleted by mistake

Reply:
I'll get it back, thanks for pointing that out!

Reply:
Done